modules/common/css/lib/variables.scss

```diff
@@ -1,6 +1,8 @@
-$wdc-form-error-icon-size: 20px;
+$wdc-form-error-icon-size: 24px;
 $wdc-form-error-spacing: $wdc-spacing-xxs;
 $wdc-form-label-line-height: 16px;
-$wdc-form-inline-label-width: 176px;
+$wdc-form-inline-label-width: 180px;
 $wdc-form-field-min-width: 280px;
+$wdc-form-textarea-min-height: $wdc-spacing-xxl;
+$wdc-form-selection-control-spacing: 18px;
```

Comment:
Since this is the width/height of checkboxes and radios, I would call this the size, not the spacing.